shards/modules/physics/constraints.cpp

```diff
@@ -343,31 +341,31 @@ struct DistanceConstraint final : public ConstraintBase {
-  static SHOptionalString help() { return SHCCSTR(""); }
+  static SHOptionalString help() { return SHCCSTR("This shard creates a slider constraint between two bodies, turning the two bodies into sliders that can slide along the axis specified in the SliderAxis parameter."); }
 
   PARAM_VAR(_space, "Space",
             "This determines in which space the constraint is setup, all other properties should be in the specified space",
             {ConstraintSpaceEnumInfo::Type});
-  PARAM_PARAMVAR(_refA, "FirstPoint", "Body 1 constraint reference frame (space determined by Space)",
+  PARAM_PARAMVAR(_refA, "FirstPoint", "The position of the connection point for the first body.",
                  {CoreInfo::Float3Type, CoreInfo::Float3VarType});
-  PARAM_PARAMVAR(_refB, "SecondPoint", "Body 2 constraint reference frame (space determined by Space)",
+  PARAM_PARAMVAR(_refB, "SecondPoint", "The position of the connection point for the second body.",
                  {CoreInfo::Float3Type, CoreInfo::Float3VarType});
 
-  PARAM_PARAMVAR(_sliderAxis, "SliderAxis", "Axis along which movement is possible (direction)",
+  PARAM_PARAMVAR(_sliderAxis, "SliderAxis", "Axis along which movement is possible.",
                  {CoreInfo::Float3Type, CoreInfo::Float3VarType});
-  PARAM_PARAMVAR(_normalAxis, "NormalAxis", "Perpendicular vector to define the frame",
+  PARAM_PARAMVAR(_normalAxis, "NormalAxis", "Vector perpendicular to the slider axis, to define the frame",
                  {CoreInfo::Float3Type, CoreInfo::Float3VarType});
 
-  PARAM_PARAMVAR(_limitsMin, "LimitsMin", "Minimum limit of movement", {CoreInfo::FloatType, CoreInfo::FloatVarType});
-  PARAM_PARAMVAR(_limitsMax, "LimitsMax", "Maximum limit of movement", {CoreInfo::FloatType, CoreInfo::FloatVarType});
+  PARAM_PARAMVAR(_limitsMin, "LimitsMin", "The maximum distance the slider can move in the negative direction.", {CoreInfo::FloatType, CoreInfo::FloatVarType});
+  PARAM_PARAMVAR(_limitsMax, "LimitsMax", "The maximum distance the slider can move in the positive direction.", {CoreInfo::FloatType, CoreInfo::FloatVarType});
 
-  PARAM_PARAMVAR(_limitSpring, "LimitSpring", "When set, makes the limits soft", SpringSettings::TypesVarOrNone);
+  PARAM_PARAMVAR(_limitSpring, "LimitSpring", "When set, makes the limits soft.", SpringSettings::TypesVarOrNone);
 
   PARAM_PARAMVAR(_maxFrictionForce, "MaxFrictionForce",
-                 "Maximum amount of friction force to apply (N) when not driven by a motor",
+                 "Maximum amount of friction force to apply (in Newtons) on the slider.",
                  {CoreInfo::FloatType, CoreInfo::FloatVarType});
 
-  PARAM_PARAMVAR(_motorSettings, "MotorSettings", "Motor settings around the sliding axis", {MotorSettings::TypesVarOrNone});
+  PARAM_PARAMVAR(_motorSettings, "MotorSettings", "Defines the maximum and minimum amount of force applied to the slider as it slides along the axis.", {MotorSettings::TypesVarOrNone});
```

Comment:
This is meant for motors, where the body is driven by a "motor" along the slider. Although I am still missing the part that actually enables this, so it's for future use.